src/lib.rs

```diff
@@ -254,6 +254,10 @@ impl Client {
     /// On Unix and Windows this will clobber the `CARGO_MAKEFLAGS` environment
     /// variables for the child process, and on Unix this will also allow the
     /// two file descriptors for this client to be inherited to the child.
+    ///
+    /// This method doesn't exist on platforms other than Unix and Windows, as
+    /// those don't have a real jobserver implementation.
+    #[cfg(any(unix, windows))]
```

Comment:
Isn't it better to leave the method for API compatibility, but make it no-op in the `imp` itself? (like it is in Windows case)

Reply:
Done